index.html

```diff
@@ -117,7 +118,7 @@ <h2 class="my-recent-work" id="my-recent-work">MY RECENT WORK</h2>
 
       <section class ="work-grid">
```

Comment:
- [ ] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads), you can change your existing code by removing already created HTML code for the projects list and instead use JavaScript to create this section

Reply:
As it is a project requirement and we are here to learn, now is the best time to practice our skills. We need to refactor our code for this milestone in order for the work projects to be rendered from a JavaScript array of objects.

- [ ] Kindly, update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all HTML in that section is created when the page loads. :100:

Reply:
@Ismailco Sorry, I don't understand what is wrong. The projects informations are already being stored and shown with JavaScript

Reply:
@Gopxfs, The HTML should generate dynamically using JavaScript, you are creating this section using static HTML